src/js/users/queries.ts

```diff
@@ -37,3 +39,11 @@ export function useInfiniteFindUsers(per_page: number, term: string) {
         },
     );
 }
+
+export function useCreateFirstUser() {
+    return useMutation<User, ErrorResponse, NewUser>(createFirst, {
+        onSuccess: () => {
```

Comment:
```suggestion
export function useCreateFirstUser() {
    return useMutation<User, ErrorResponse, {handle: string, password: string, forceReset: boolean}>(({handle, password, forceReset}) => createFirst(handle, password, forceReset), {
        onSuccess: () => {
```

Everything looks good but just a few nitpicks: 
- I would do this so it can stay consistent with the other mutations we have. 
- Also you may want to destructure the props for createFirst from `({handle, password, forceReset})` to `(handle: string, password: string, forceReset: boolean)`
- Then you can probably delete the `NewUser` type from types

Reply:
I made these changes, but I also had to add the `force_reset` param to the api call in `createFirst`. Tests still pass but let me know if that won't work.

Reply:
Yeah good call.